packages/components/form-field/src/FormFieldLabel.tsx

```diff
@@ -38,7 +38,6 @@ export const FormFieldLabel = forwardRef<HTMLLabelElement, FormFieldLabelProps>(
         htmlFor={htmlFor}
         className={cx(
           className,
-          'flex items-center gap-sm',
```

Comment:
@acd02 I'm afraid removing this could visually break custom labels on Polaris side, wdyt ?

(But to do this ticket we should remove `flex` behaviour from the label.

Reply:
There is only one occurrence in the Polaris codebase where a **flex-related** class is passed to the `FormField.Label`, so I think we're safe.

- src/components/InputPhone/wrapper.tsx